apps/client/src/pages/home/components/performance-carousel-section.tsx

```diff
@@ -1,24 +1,26 @@
 import { PerformanceCarousel } from '@confeti/design-system';
 
 import { useNavigateToDetail } from '@shared/hooks/use-navigate-to-detail';
-import { CarouselPerformances } from '@shared/types/home-response';
-import { checkIsNotLoggedIn } from '@shared/utils/check-is-not-logged-in';
+import { CarouselPerformancesResponse } from '@shared/types/home-response';
 import { convertToCdnUrl } from '@shared/utils/convert-to-cdn-url';
 import { formatDate } from '@shared/utils/format-date';
 
 import * as styles from './performance-carousel-section.css';
 
 const PerformanceCarouselSection = ({
-  data,
+  latestPerformances,
 }: {
-  data: CarouselPerformances[];
+  latestPerformances: CarouselPerformancesResponse;
 }) => {
   const navigateToDetail = useNavigateToDetail();
-  const badgeText = checkIsNotLoggedIn()
-    ? '다가오는 공연'
-    : '선호하는 아티스트';
 
-  const displayPerformances = data.length > 7 ? data.slice(0, 7) : data;
+  const badgeText = latestPerformances.isPersonalized
+    ? '선호하는 아티스트'
+    : '다가오는 공연';
+  const displayPerformances =
+    latestPerformances.performances.length > 7
+      ? latestPerformances.performances.slice(0, 7)
+      : latestPerformances.performances;
```

Comment:
별건 아닌데 배열의 길이가 7보다 작아도 그대로 동작해서..
```suggestion
  const displayPerformances = latestPerformances.performances.slice(0, 7);

```
조건문 없어도 될듯용

Reply:
good 감사함다